Makefile

```diff
@@ -141,8 +141,8 @@ lintfix: ## Try to automatically fix any Ruby, ERB, JavaScript, YAML, or CSS lin
 	@echo "--- normalize yaml ---"
 	make normalize_yaml
 
-brakeman: ## Runs brakeman
-	bundle exec brakeman
+brakeman: ## Runs brakeman code security check
+	(bundle exec brakeman) || (echo "Error: update code as needed to remove security issues. For known exceptions already in brakeman.ignore, use brakeman to interactively update exceptions."; exit 1)
```

Comment:
```suggestion
	(bundle exec brakeman) || (echo "Error: update code as needed to remove security issues. For known exceptions already in brakeman.ignore, use brakeman -I to interactively update exceptions."; exit 1)
```

Reply:
I have a slight preference towards not nudging towards adding new exceptions and leaving it as-is. Would that be alright?